Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -99,13 +99,18 @@ private final class TokenStreamCreator: SyntaxVisitor {
     super.visit(node)
   }
 
+  override func visit(_ node: TupleElementListSyntax) {
+    if node.count > 0 {
```

Comment:
Does this exact loop show up frequently enough that it would be useful to follow this PR up with a refactor that adds a helper method like `insertBreaksBetweenChildren(of: node)`?